### PR TITLE
SLCORE-432 Extract description sections for security hotspots

### PR DESCRIPTION
**Pull request by jblievremont**

### CODE REVIEW

File: its/tests/src/test/java/its/ConnectedModeBackendTest.java [hunk 1]
```diff
@@ -181,6 +187,174 @@ public void returnDescriptionSectionsForTaintRules() throws ExecutionException,
...
+    assumeThat(ORCHESTRATOR.getServer().version()).isGreaterThanOrEqualTo(Version.create("9.7"));
+
+    backend.getConfigurationService().didAddConfigurationScopes(new DidAddConfigurationScopesParams(
+      List.of(new ConfigurationScopeDto("project", null, true, "Project", new BindingConfigurationDto("ORCHESTRATOR", PROJECT_KEY_JAVA_HOTSPOT, false)))));
+
+    var activeRuleDetailsResponse = backend.getActiveRulesService().getActiveRuleDetails(new GetActiveRuleDetailsParams("project", javaRuleKey(ORCHESTRATOR, "S4792"))).get();
+
+    var description = activeRuleDetailsResponse.details().getDescription();
+    assertThat(description)
+      .extracting("right.introductionHtmlContent")
```
Comment (henryju) [inline its/tests/src/test/java/its/ConnectedModeBackendTest.java:201]: I think using `extracting` here is not required. Would be cleaner with `assertThat(description.getRight().getIntroductionHtmlContent())`.

File: its/tests/src/test/java/its/ConnectedModeBackendTest.java [hunk 2]
```diff
@@ -181,6 +187,174 @@ public void returnDescriptionSectionsForTaintRules() throws ExecutionException,
...
+      .isNull();
+    assertThat(description)
+      .extracting("right.tabs", as(list(ActiveRuleDescriptionTabDto.class)))
```
Comment (henryju) [inline its/tests/src/test/java/its/ConnectedModeBackendTest.java:204]: Same, I would not use `extracting` here.

**Review by damien-urruty-sonarsource — APPROVED** [review]
LGTM, my only comment is that I recently removed asserting rule description sections content in another test. The description can change relatively often so it is a bit fragile to check that

**Review by henryju — APPROVED** [review]
Minor feedback then LGTM

**Pull request merged**